src/Hyperwallet/Hyperwallet.php

```diff
@@ -1803,4 +1826,59 @@ private function updateTransferMethod($userToken, $transferMethod, $transferMeth
                 ), $transferMethod, array());
     }
 
+    /**
+     * Create a bank card status transition
+     *
+     * @param string $userToken The user token
+     * @param UserStatusTransition $transition The status transition
+     * @return UserStatusTransition
+     *
+     * @throws HyperwalletArgumentException
+     * @throws HyperwalletApiException
+     */
+
+    public function updateVerificationStatus($userToken,$verificationStatus){
```

Comment:
Format pls

Reply:
is it the blank line 1839 or the space after the comma. I did both. Let me know if there is anything else to be done.